### PR TITLE
Migrated status to CSS modules

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
* Converted CSS to be a local module
* Updated tests

@cerner/terra

### CODE REVIEW

**Comment by bjankord** [comment]
Got a lint error I need to fix.

File: packages/terra-status/src/Status.jsx [hunk 1]
```diff
@@ -16,8 +18,8 @@ const propTypes = {
 };
 
 const Status = ({ color, children, ...customProps }) => {
-  const statusClassNames = classNames(
-    'terra-Status',
+  const statusClassNames = cx(
+    'status',
     { [`${customProps.className}`]: customProps.className },
```
Comment (mhemesath) [inline packages/terra-status/src/Status.jsx]: this can just be `customProps.className`.. if it isn't present, classNames won't render it out

Comment (mhemesath) [inline packages/terra-status/src/Status.jsx]: You couldn't actually just put this inline in the jsx
```
<div {...customProps} style={{ borderColor: color }} className={cx('status', customProps.className)}>
```

Comment (bjankord) [inline packages/terra-status/src/Status.jsx]: Updated: https://github.com/cerner/terra-core/pull/592/commits/299a5aa41ab983d43a57a8a27a060bdace7d4caf

**Comment by mmalaker** [comment]
Functional verification completed. Status examples display and function correctly after conversion to CSS modules.

**Pull request merged**